create_graphics.py

```diff
@@ -447,10 +449,13 @@ def graphics_driver(cla):
     '''
 
     # Create an empty zip file
-    zipf = None
     if cla.zip_dir:
-        os.makedirs(cla.zip_dir, exist_ok=True)
-        zipf = os.path.join(cla.zip_dir, 'files.zip')
+        zipfiles = {}
+        for tile in cla.tiles:
+            tile_zip_dir = os.path.join(cla.zip_dir, tile)
+            os.makedirs(tile_zip_dir, exist_ok=True)
+            tile_zip_file = os.path.join(tile_zip_dir, 'files.zip')
+            zipfiles[tile] = tile_zip_file
```

Comment:
Consider combining lines 457 and 458 for efficiency and brevity.

Reply:
Thanks!  I don't know why I didn't see that.